roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1278,12 +1278,20 @@ def get_docker_version_info():
     """ Parses and returns the docker version info """
     result = None
     if is_service_running('docker'):
-        version_info = yaml.safe_load(get_version_output('/usr/bin/docker', 'version'))
-        if 'Server' in version_info:
-            result = {
-                'api_version': version_info['Server']['API version'],
-                'version': version_info['Server']['Version']
-            }
+        try:
+            version_info = yaml.safe_load(get_version_output('/usr/bin/docker', 'version'))
+            if 'Server' in version_info:
+                result = {
+                    'api_version': version_info['Server']['API version'],
+                    'version': version_info['Server']['Version']
+                }
+        # PyYAML unable to parse the output of `docker version`
+        except yaml.scanner.ScannerError:
+            raise OpenShiftFactsInternalError('Unable to determine the installed docker version.'
+                                              ' Check the output of `docker version` and ensure that the'
+                                              ' correct version of docker has been installed per the'
+                                              ' host preparation steps in the OpenShift documentation'
```

Comment:
Alternatively/in-concert, can you include the actually read output from the command that errored (include the result from the docker version call)